docker-compose.yml

```diff
@@ -40,22 +40,22 @@ services:
     #     ENCRYPT_KEY: "${ENCRYPT_KEY}"
 
 
-    web_app:
-      build:
-        context: roadside-forms-frontend/frontend_web_app
-        dockerfile: Dockerfile-local
-        args: 
-          REACT_APP_BASE_URL: "${REACT_APP_BASE_URL_DEV}"
-          REACT_APP_KEYCLOAK_URL: "${KEYCLOAK_AUTH_URL}"
-          REACT_APP_KEYCLOAK_REALM: "${KEYCLOAK_REALM}"
-          REACT_APP_KEYCLOAK_CLIENT_ID: "${KEYCLOAK_CLIENT_ID}"
-          REACT_APP_API_BASE_URL: "${REACT_APP_API_BASE_URL}"
-      depends_on:
-        - roadside-forms-api
-      networks:
-        - docker-network
-      ports:
-        - "8080:5000"
+    # web_app:
```

Comment:
is this change needed?

Reply:
I just commented it out cause I only use it when testing specific functionality. I run NPM to host the app when I am developing for hot reload.

Reply:
Personally, I also use NPM for frontend but I think we should not commit it as commented out code as it will be good for someone new to the app to run it with docker as a complete app instead of trying to figure out how to run frontend separately. my two cents.